src/HasTranslations.php

```diff
@@ -185,9 +192,16 @@ protected function normalizeLocale(string $key, string $locale, bool $useFallbac
         return $locale;
     }
 
-    protected function getLocale(): string
+    public function setLocale(string $locale): self
+    {
+        $this->locale = $locale;
```

Comment:
I like the functionality but have one concern. What happens if the model has a `locale` attribute of its own? Maybe we just a name that has less change of colliding with an existing attribute.

Reply:
Good point, I changed it to `$translationLocale`.

In the event that someone has a column name or property on his model that has the same name, It will override the trait property.